### ISSUE TITLE
PHP7 Compatibility issues multiple files in Pods Core

### ISSUE THREAD

**Issue by PodsBot** [pods-framework/pods#3828]
PHP7 Compatibility issues multiple files in Pods Core submitted via Slack by jimtrue

Referenced from this Forum post:
https://wordpress.org/support/topic/php7-error-messages/
- [x] classes/PodsData.php - Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead. - Lines 2141, 2145, 2199, 2203
- [x] components/Builder/Modules issues in multiple files - Use of deprecated PHP4 style class constructor is not supported since PHP 7 (field/PodsBuilderModuleField.php, line 21, form/PodsBuilderModuleForm.php, line 21, list/PodsBuilderModuleList.php, line 21, single/PodsBuilderModuleSingle.php, line 21, view/PodsBuilderModuleView.php, line 21)
- [x] components/Markdown.php - Use of deprecated PHP4 style class constructor is not supported since PHP 7, line 230
- [x] deprecated/deprecated.php - Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead. - Lines 89, 91, 93, 101

**Comment by jimtrue** [comment]
From this plugin checker: PHP linter: https://wordpress.org/plugins/php-compatibility-checker/

```
FILE: /home/www/wp-content/plugins/pods/classes/PodsData.php
---------------------------------------------------------------------------------------------------------------
FOUND 4 ERRORS AFFECTING 4 LINES
---------------------------------------------------------------------------------------------------------------
 2141 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 2145 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 2199 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 2203 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
---------------------------------------------------------------------------------------------------------------


FILE: /home/www/wp-content/plugins/pods/components/Builder/modules/field/PodsBuilderModuleField.php
---------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------
 21 | ERROR | Use of deprecated PHP4 style class constructor is not supported since PHP 7
---------------------------------------------------------------------------------------------------


FILE: /home/www/wp-content/plugins/pods/components/Builder/modules/form/PodsBuilderModuleForm.php
-------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------
 21 | ERROR | Use of deprecated PHP4 style class constructor is not supported since PHP 7
-------------------------------------------------------------------------------------------------


FILE: /home/www/wp-content/plugins/pods/components/Builder/modules/list/PodsBuilderModuleList.php
-------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------
 21 | ERROR | Use of deprecated PHP4 style class constructor is not supported since PHP 7
-------------------------------------------------------------------------------------------------


FILE: /home/www/wp-content/plugins/pods/components/Builder/modules/single/PodsBuilderModuleSingle.php
-----------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------
 21 | ERROR | Use of deprecated PHP4 style class constructor is not supported since PHP 7
-----------------------------------------------------------------------------------------------------


FILE: /home/www/wp-content/plugins/pods/components/Builder/modules/view/PodsBuilderModuleView.php
-------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------
 21 | ERROR | Use of deprecated PHP4 style class constructor is not supported since PHP 7
-------------------------------------------------------------------------------------------------


FILE: /home/www/wp-content/plugins/pods/components/Markdown.php
-------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------
 230 | ERROR | Use of deprecated PHP4 style class constructor is not supported since PHP 7
-------------------------------------------------------------------------------------------


FILE: /home/www/wp-content/plugins/pods/deprecated/deprecated.php
--------------------------------------------------------------------------------------------------------------
FOUND 5 ERRORS AFFECTING 4 LINES
--------------------------------------------------------------------------------------------------------------
  89 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
  89 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
  91 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
  93 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 101 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
--------------------------------------------------------------------------------------------------------------
```

**Comment by JoryHogeveen** [comment]
Doing a PR today.

One problem with the `pod_query` function in deprecated.php. `$wpdb->dbh` is a protected property and not allowed. How did this even work?!

**Comment by Ramoonus** [comment]
@jimtrue what tool is that? 
could you do it for php 7.1 too?

**Comment by sc0ttkclark** [comment]
Fixed via #3830, php 7.1 can be tackled in #3920
